src/fabric/src/fabric2_util.erl

```diff
@@ -124,6 +126,20 @@ validate_json_list_of_strings(Member, Props) ->
     end.
 
 
+hash_atts([]) ->
+    <<>>;
+
+hash_atts(Atts) ->
+    lists:sort(fun(A, B) ->
```

Comment:
Am I reading it correctly we ignore the result of a lists:sort?

Reply:
You sure are! Fixed now.